.aws/task_definition_prod_worker.json

```diff
@@ -18,28 +18,29 @@
             "environment": [],
             "environmentFiles": [
                 {
-                    "value": "arn:aws:s3:::quivr-env-variables/production.env",
+                    "value": "arn:aws:s3:::sr-vt-uw1-prod-env-bucket/production.env",
                     "type": "s3"
                 }
             ],
             "mountPoints": [],
             "volumesFrom": [],
+            "readonlyRootFilesystem": false,
             "logConfiguration": {
                 "logDriver": "awslogs",
                 "options": {
                     "awslogs-create-group": "true",
-                    "awslogs-group": "/ecs/quivr-production-chat",
-                    "awslogs-region": "eu-west-3",
-                    "awslogs-stream-prefix": "ecs"
+                    "awslogs-group": "/ecs/sr-vt-prod-worker",
+                    "awslogs-region": "us-west-1",
+                    "awslogs-stream-prefix": "sr-vt-prod-worker"
                 }
             }
         }
     ],
-    "family": "quivr-prod-chat",
-    "taskRoleArn": "arn:aws:iam::253053805092:role/ecsTaskExecutionRole",
-    "executionRoleArn": "arn:aws:iam::253053805092:role/ecsTaskExecutionRole",
+    "family": "sr-vt-prod-worker",
+    "taskRoleArn": "arn:aws:iam::155722108667:role/sr-vt-prod-worker-task",
```

Comment:
can we use arn:aws:iam::155722108667:role/sr-vt-github-actions for now?
do we need to create all these role as such a early stage or a project?

Reply:
I am fine with having all these roles, if we are following the best practise. 
but I only tested `sr-vt-github-actions` role.
so if we decide to do it differently for production, we have to test the workflow a couple times at least. 
to make sure it works.

Reply:
This is following best practices to keep iam roles with least privileges in prod envs.

Later we will revisit the iam role with admin rights for preview env to secure it, so we would not affect the prod env.